.clang-tidy

```diff
@@ -13,8 +13,12 @@ Checks: -*,
   -bugprone-exception-escape,
   -cppcoreguidelines-avoid-magic-numbers,
   -cppcoreguidelines-non-private-member-variables-in-classes,
+  -cppcoreguidelines-pro-bounds-constant-array-index,
+  -cppcoreguidelines-pro-bounds-pointer-arithmetic,
   -google-readability-casting,
+  -google-default-arguments,
```

Comment:
There is a core class that uses a default argument that a ton of stuff inherits from (the `print` method in fuse models) so changing all these would be annoying.